DuckDuckGo/TabManager.swift

```diff
@@ -47,11 +47,8 @@ class TabManager {
         let contentBlocker = ContentBlockerConfigurationUserDefaults()
         let configuration =  WKWebViewConfiguration.persistent()
         let controller = TabViewController.loadFromStoryboard(model: tab, contentBlocker: contentBlocker)
-        controller.attachWebView(configuration: configuration)
+        controller.attachWebView(configuration: configuration, andLoadUrl: url)
```

Comment:
There's a completion handler in `attachWebView` which reattaches DDG cookies and that loads the URL. This fixes duplicate atb calls.